.idea/.generators

```diff
@@ -5,4 +5,4 @@ You are allowed to:
 2. Remove generators
```

Comment:
Are these `.idea/*` files auto-generated?  Could you remove them and add the following to the `.gitignore` file?  
```
.idea
.idea/*
```